cmd/spire-agent/cli/run/run.go

```diff
@@ -176,6 +177,7 @@ func parseFlags(args []string) (*agentConfig, error) {
 	flags.StringVar(&c.SocketPath, "socketPath", "", "Location to bind the workload API socket")
 	flags.StringVar(&c.TrustDomain, "trustDomain", "", "The trust domain that this agent belongs to")
 	flags.StringVar(&c.TrustBundlePath, "trustBundle", "", "Path to the SPIRE server CA bundle")
+	flags.BoolVar(&c.InsecureBootstrap, "insecureBootstrap", false, "If true, the agent bootstraps without verifying the server's identity")
```

Comment:
nit: s/If true/If set/

Does this work if I do something like `-insecureBootstrap true`?

Reply:
If you think that sounds clearer, i'm fine changing it. This flag can be "set" to false though, (e.g. `-insecureBootstrap=false`)

```
foo$ cat main.go
package main

import (
	"flag"
	"fmt"
)

func main() {
	boolFlag := flag.Bool("bool", false, "blah")
	flag.Parse()
	fmt.Println(*boolFlag)
}
foo$ go build && ./foo
false
foo$ go build && ./foo -bool=false
false
foo$ go build && ./foo -bool=true
true
```